CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 1.17.1 (2020-01-13)
+
+This is a minor security patch release fixing a problem with the scoping
+of `imagePullSecret`s and removing git-URL HTTPS credentials server-side.
+
+### Fixes
+
+- Correctly scope imagePullSecrets by their namespace [fluxcd/flux#2728][]
+- Sanitize Git remote URLs on the server side [fluxcd/flux#2726][]
+
+[fluxcd/flux#2726]: https://github.com/fluxcd/flux/pull/2726
+[fluxcd/flux#2728]: https://github.com/fluxcd/flux/pull/2728
+
```

Comment:
We are forgetting to thank ourselves here.

In addition, I just noticed there is a typo in the previous changelog, `Enhacements` -> `Enhancements`. Is this typo part of the release utility that was introduced or just a generic typo?

Reply:
I intentionally omitted the acknowledgement since I don't need to thank myself :)

I will look into the typo, I think it's the tool.

Reply:
The thank you note usually also includes the person who reported the issue, and the reviewer (although the latter would be me, and I can live without the credit), so although small and mostly us, I would still include it.

Reply:
> The thank you note usually also includes the person who reported the issue, and the reviewer

Ah, really? Then we should update the CHANGELOG generator, because it currently only includes the PR creators.